src/components/issues/ActionForm.tsx

```diff
@@ -135,10 +137,18 @@ const ActionForm: FC<ActionFormProps> = ({
                                 : styles.suggestions_container
                         }
                     >
+                        <label
```

Comment:
Should there be little space between label and input? Currently they are touching each other.

Reply:
Are you referring to code or UI?

Reply:
Indentation is taken care by prettier.

Reply:
I was referring to UI. The 'g' in 'Assignee' label seems to be touching the input in screenshot?

Reply:
Added slight gap.